### ISSUE TITLE
eia930 - Investigate shrinkages in every single file except 2024half2.

### ISSUE THREAD

**Issue by e-belfer** [catalyst-cooperative/pudl-archiver#493]
In the December 1st archive we see shrinkages in every file except the latest half-year. This isn't a compression change, but an actual change in the .txt file. It wasn't large enough to trip the validation tests, but we should investigate this change before we publish this archive.

**Comment by cmgosnell** [comment]
Okay, I have determined why the new files are smaller and its generally good news.

TL;DR: The old archives store integer values over 3 digits with quotes and commas. ex: 1,072 is stored as `"1,072"` in the old files and stored as `1072` in the new files.

## How I determined this?
* poked around at size and shape of new vs old files. seemed like there are no record losses per some spot checks.
* loaded them both into pandas dfs. used the `compare` method. noticed dtypes were off. 
* tried `compare` after `convert_dtypes`. dtypes were still off.
* post `convert_dtypes`, took one of the still object columns and applied `astype(pd.Int64Dtype())`.. as expected got an error: `invalid literal for int() with base 10: '1,072'`
* opened the CSV version of both files to verify

### next steps
I'm going to publish this [draft archive](https://zenodo.org/uploads/14252683)! we should run it through pudl to see if its does all the things but that should happen in the new year with the full quarterly updates presumably 

@e-belfer does that all sounds right and good?
